services/frontend/config/init.sh

```diff
@@ -3,7 +3,9 @@
 
 apk update && apk add jq
 
-[ "$BE_VERSION" = "v4" ] && EXCLUDE="[!v3]"
+INCLUDE="base"
```

Comment:
Could this have used GLOBIGNORE (appending all exclusions to it). If we later decide to split the config.json based on functionality, I think excluding will work better

Reply:
Unfortunately, `GLOBIGNORE` is bash only